src/app/welcome/welcome.component.html

```diff
@@ -38,15 +38,15 @@
           <div class="uk-card-body">
 
             <ul uk-accordion="">
-              <li class="uk-open">
```

Comment:
We should keep it open until we redesign the homepage.

Reply:
Restored